homeassistant/components/goalzero/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "Goal Zero Yeti",
```

Comment:
This title doesn't need translation (as it is a brand?) Thus should not be in the language file.

```suggestion
```